contract_wrapper/KlerosWrapper.js

```diff
@@ -30,22 +30,45 @@ class KlerosWrapper extends ContractWrapper {
    * @return  truffle-contract Object | err The contract object or error deploy
    */
   deploy = async (
+      rngAddress = '0xB7669d5774Cc4E33a056B2A24A7c7B7957556b37',
```

Comment:
I think it is better in the `.env.prod` and `.env.dev` files

Reply:
I will just delete these